.github/labeler.yml

```diff
@@ -43,16 +47,19 @@ circulation:
   - rero_ils/modules/items/utils.py
   - rero_ils/modules/loans/**/*
   - rero_ils/modules/ill_requests/**/*
+  - rero_ils/modules/patron_types/**/*
 
 # Add 'user management' label to any changes in users or patrons folders and
 # subfolders
 "user management":
   - rero_ils/modules/users/**/*
   - rero_ils/modules/patrons/**/*
+  - rero_ils/modules/patron_types/**/*
 
 # Add acquisition label to any changes in acquisition related folders and
 # subfolders
 acquisition:
+  - rero_ils/acquisition/**/*
```

Comment:
Can not find a directory `rero_ils/acquisition` in project? Maybe `rero_ils/modules/acq_orders/**/*`

Reply:
@rerowep You're right, it's coming: https://github.com/rero/rero-ils/pull/1948